CalcualtorLogic.js

```diff
@@ -106,12 +106,11 @@ class Calculator {
         this.resultElement.textContent = this.currentResult;
     }
     onOperatorButtonClick(event) {
-        const curntRsltAndOperclkdBtn = this.currentResult + event.target.dataset.operator;
-        this.currentResult = curntRsltAndOperclkdBtn;
-        this.updateResultElement();
+        const curntRsltAndOperclkdBtn = event.target.dataset.operator;
```

Comment:
what you explain what are you storing in `curntRsltAndOperclkdBtn`

Reply:
the clicked operator (+||-||*||/)

Reply:
In this case can you change the name of the variable to something more related to what you are planning to store it?
I don't understand why there's `curntRslt` and `And` in the name of the variable